src/lookup.py

```diff
@@ -1,7 +1,37 @@
 import xml.etree.cElementTree as ET
+from xml.dom import minidom
 import os, abc, re
 
 from itertools import zip_longest
+from PyQt5.QtGui import QImage
+
+import src.anm2 as anm2
+from src.constants import *
+
+
+def printf(*args):
```

Comment:
violates DRY, maybe we should make a "src.util" file and stick this in there, and then have everything import printf from that, thoughts?

Reply:
yeah agreed something like that would be best